src/AppBundle/Resources/translations/messages.en.yml

```diff
@@ -0,0 +1,2 @@
+sylius_demo:
+    verification_link_flash: 'To verify your account - please visit %url%'
```

Comment:
`For demo purposes you can visit %url% to verify the account` - to indicate this flash is visible only on demo.